README.md

```diff
@@ -41,12 +42,110 @@ This plugin can be consumed by the CAP application deployed on BTP to store thei
 >
 > A reference to adding this can be found [here](https://github.com/cap-java/sdm/blob/691c329f4c3c17ae390cfcb2db1ef02650585aee/cap-notebook/demoapp/pom.xml#L20)
 
-## Deploying and testing the application
+## Setup
 
-1. Log in to Cloud Foundry space:
+In this guide, we use the Bookshop sample app in the [deploy branch](https://github.com/cap-java/sdm/tree/deploy) of this repository, to integrate SDM CAP plugin.  Follow the steps in this section for a quick way to deploy and test the plugin without needing to create your own custom CAP application.
 
-   ```sh
+### Using the released version
+If you want to use the version of SDM CAP plugin released on the central maven repository follow the below steps:
+
+1. Remove the sdm and sdm-root folders from your local .m2 repository. This ensures that the CAP application uses the plugin version from the central Maven repository, as the local .m2 repository is prioritized during the build process.
+
+2. Clone the sdm repository:
+
+```sh
+   git clone https://github.com/cap-java/sdm
+```
+
+3. Checkout to the branch **deploy**:
+
+```sh
+   git checkout deploy
+```
+
+4. Navigate to the demoapp folder:
+
+```sh
+   cd cap-notebook/demoapp
```

Comment:
Where are you specifying to add to pom dependency

Reply:
Added in another section